RecoLocalTracker/SiPixelClusterizer/plugins/alpaka/ClusterChargeCut.h

```diff
@@ -111,9 +125,18 @@ namespace pixelClustering {
         for (auto i : cms::alpakatools::independent_group_elements(acc, nclus)) {
           newclusId[i] = ok[i] = (charge[i] >= chargeCut) ? 1 : 0;
           if (0 == ok[i])
-            good = false;
+            good &= false;  //better than simple assignment in case of race?
```

Comment:
No, I think that a simple assignment is better, because it does not need to read the old value

Reply:
Understood, reverting.

Reply:
About the "in case of race", the `good` is a local variable, and therefore there should be no data races with it.